src/scss/skin-super-modern/components/_adskipbutton.scss

```diff
@@ -0,0 +1,39 @@
+@import '../variables';
+
+%ui-button-ad-skip {
+  @extend %ui-button;
+
+  background-color: transparentize(#000, 0.75);
+  border-radius: 20px;
+  padding: 0em 1em;
+  min-width: fit-content;
+
+  .#{$prefix}-label {
+    display: inline-block;
+    color: $color-ads;
```

Comment:
TBH i find the yellow text hard to read, are we sure we want to stick with that yellow?

Reply:
Noted. I'll take that feedback back to the design process.

Reply:
Don't your eyes hurt when looking at the above screenshot and trying to figure out how may seconds there are left until skip?